frontend/src/store/middlewares/error-handler.ts

```diff
@@ -0,0 +1,16 @@
+import { Middleware } from '@reduxjs/toolkit';
```

Comment:
move to `frontend/src/store/middlewares/handle-error/handle-error.middleware.ts`